deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/BaseRecurrentLayer.java

```diff
@@ -14,6 +14,16 @@ protected BaseRecurrentLayer(Builder builder) {
         super(builder);
     }
 
+    protected boolean useLayerNormalization;
```

Comment:
Really minor style nitpick: we've been using the following order convention: fields -> constructors -> methods. Move it to the top of the class?